src/core/amount.js

```diff
@@ -105,6 +105,19 @@ Amount.NaN = function() {
   return result;                      // but let's be careful
 };
 
+Amount.createFast = function(value: Value, currency: Currency, issuer: UInt160,
```

Comment:
I wanted to emphasis somehow that it is not regular function that can be used anywhere because I'm skipping call to `_check_limits` here.
Maybe `from_components_unsafe`?